desktop/cmp/appbar/AppBar.js

```diff
@@ -86,5 +94,8 @@ AppBar.propTypes = {
     hideAppMenuButton: PT.bool,
 
     /** Options to pass to the AppMenuButton. */
-    appMenuButtonOptions: PT.object
+    appMenuButtonOptions: PT.object,
```

Comment:
Arg - this is pre-existing and not the direct target for this change, but let's take the opportunity to improve....  Can we please also make the breaking changes to:
* Rename `appMenuButtonOptions` -> `appMenuButtonProps` - this is more clear and matches what we have on mobile.
* Ensure that `AppMenuButton` takes all Button props in addition to its own special ones.  It should extract its special props and then pass any others onto the button target for its popover.  I wanted to set a custom icon for the button but could not.  (Specifically I wanted to take one of the example Toolbox apps and drop its primary app icon, replace by moving appButton to left and using the app-specific icon there instead.)
  * Note you can spread `Button.propTypes` onto `AppButton.propTypes` in terms of documenting that it takes any valid button prop.